aptos-move/framework/aptos-framework/sources/primary_fungible_store.move

```diff
@@ -104,6 +104,24 @@ module aptos_framework::primary_fungible_store {
         fungible_asset::store_exists(primary_store_address(account, metadata))
     }
 
+    
+    /// Get the address of the primary store for the given account.
+    public inline fun primary_store_address_inlined<T: key>(owner: address, metadata: Object<T>): address {
+        let metadata_addr = object::object_address(&metadata);
+        object::create_user_derived_object_address(owner, metadata_addr)
+    }
+
+    /// Get the primary store object for the given account.
+    public inline fun primary_store_inlined<T: key>(owner: address, metadata: Object<T>): Object<FungibleStore> {
+        let store = primary_store_address_inlined(owner, metadata);
+        object::address_to_object<FungibleStore>(store)
```

Comment:
nit: < FungibleStore > type annotation is not required as it can be inferred from the return type